src/databricks/labs/ucx/assessment/export.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from pathlib import Path
+
+from databricks.labs.blueprint.tui import Prompts
+
+from databricks.labs.ucx.config import WorkspaceConfig
+from databricks.labs.lsql.backends import SqlBackend
+from databricks.labs.lsql.dashboards import DashboardMetadata
+
+logger = logging.getLogger(__name__)
+
+
+class AssessmentExporter:
+    # File and Path Constants
+    _EXPORT_FILE_NAME = "ucx_assessment_results.zip"
+
+    def __init__(self, sql_backend: SqlBackend, config: WorkspaceConfig):
+        self._sql_backend = sql_backend
+        self._config = config
+
+    def export_results(self, prompts: Prompts):
+        """Main method to export results to CSV files inside a ZIP archive."""
+        valid_queries = {"azure", "estimates", "interactive", "main"}
```

Comment:
Could you make this dynamic based on the names of the subdirectories in the assessment queries?

Reply:
Updated.